delfin/drivers/dell_emc/vmax/client.py

```diff
@@ -50,7 +50,32 @@ def init_connection(self, access_info):
         self.array_id = access_info.get('extra_attributes', {}). \
             get('array_id', None)
         if not self.array_id:
-            raise exception.InvalidInput('Input array_id is missing')
+            """ Unisphere  managing  only one array (embedded unisphere)
+             can be supported with out array_id in input """
+
+            # Todo : Remove this temporary connection
+            #  while removing PyU4V dependency
+            try:
```

Comment:
Are we going to keep both of the connection mechanism, Python client as well as REST ?

Reply:
No. We will remove PyU4v client

Reply:
Ok, then this PR need to be updated once the other PR for supporting PR is merged